docker-client/1.2.2.wso2v3/pom.xml

```diff
@@ -0,0 +1,123 @@
+            <optional>true</optional>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.4.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-Vendor>WSO2, Inc.</Bundle-Vendor>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            io.fabric8.docker.*;version="${project.version}";-split-package:=merge-first
+                        </Export-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Import-Package>
+                            !io.fabric8.docker.*,
+                            com.fasterxml.jackson.annotation;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.databind;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.databind.type;version="[2.7.0,3.0.0)",
+                            okhttp3;version="[2.7.2,4.0.0)",
+                            okhttp3.logging;version="[2.7.2,4.0.0)",
+                            okhttp3.ws;version="[2.7.2,4.0.0)",
+                            io.fabric8.docker.api.builder;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.api.model;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.container;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.image;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.misc;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.network;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.volume;version="[1.0.0,2.0.0)",
+                            javax.net;version="[0.0.0,1.0.0)",
+                            javax.net.ssl;version="[0.0.0,1.0.0)",
+                            javax.security.auth.x500;version="[0.0.0,1.0.0)",
+                            javax.validation;version="[1.1.0,2.0.0)",
+                            okio;version="[1.6.0,2.0.0)",
+                            org.apache.commons.codec.binary;version="[1.4.0, 1.5.0)",
+                            org.apache.commons.compress.archivers;version="[1.9.0,2.0.0)",
+                            org.apache.commons.compress.archivers.tar;version="[1.9.0,2.0.0)",
+                            org.apache.commons.compress.compressors.bzip2;version="[1.9.0,2.0.0)",
+                            org.newsclub.net.unix;version="[2.0.0,2.1.0)",
+                            org.slf4j;version="[1.7.0,2.0.0)"
```

Comment:
Any reason why we are not adding resolution:=optional here?

I.E. : https://github.com/wso2/orbit/blob/master/chill/0.5.0.wso2v1/pom.xml#L105